sdk/build.gradle

```diff
@@ -81,73 +88,22 @@ artifacts {
     archives androidSourcesJar, androidJavadocsJar
 }
 
-publishing {
```

Comment:
Moving this to buildSrc will simplify the publication process of add-on sdks, such as separate ktx publications.